src/main/java/de/tum/in/www1/artemis/domain/quiz/ShortAnswerSubmittedText.java

```diff
@@ -80,7 +80,11 @@ public void setSubmittedAnswer(ShortAnswerSubmittedAnswer shortAnswerSubmittedAn
      * @return boolean true if submittedText fits the restrictions above, false when not
      */
     public boolean isSubmittedTextCorrect(String submittedText, String solution) {
-        return FuzzySearch.ratio(submittedText.toLowerCase().trim(), solution.toLowerCase().trim()) > 85;
+        ShortAnswerQuestion saQuestion = ((ShortAnswerQuestion) submittedAnswer.getQuizQuestion());
+        if (saQuestion.matchLetterCase()) {
+            return FuzzySearch.ratio(submittedText.trim(), solution.trim()) >= saQuestion.getSimilarityValue();
```

Comment:
we should also have a fallback case here for all questions or questions in which these values are not specified (for whatever reason). The fall back for null values should be
saQuestion.matchLetterCase(): false
saQuestion.getSimilarityValue(): 85